### ISSUE TITLE
Official support for Dell Venue 8 (Windows 8 tablet)

### ISSUE THREAD

**Issue by pjcozzi** [AnalyticalGraphicsInc/cesium#1847]
Initial results on the Dell Venue 8 with IE 11 DC1 are very promising.  Cesium Viewer runs at 25 FPS in full-screen.

Issues:

* [x] Only touch drag to rotate works.  Touch for zoom and tilt do not work.
* [ ] Double tap doesn't zoom to objects in the Sandcastle CZML example.
* [x] Sandcastle (probably all one issue; everything is fine in full-screen)
   * [x] Pinch-zoom scales the entire webpage.
   * [x] Touch drag doesn't work.
   * [x] Touch doesn't work with Cesium UI elements (but Dojo/HTML5 UI elements work and Cesium picking works).
* [ ] Tests failures are the same as desktop IE, except `Renderer/Texture can create a texture from the framebuffer` fails sometimes (we can just disable it if needed).

CC #1616 (desktop IE 11 support).

**Comment by mramato** [comment]
I vote to close this issue, @shunter implemented Pointer Events for IE and I just went through and tested everything and checked off the ones that worked, above.  Since double tap doesn't work on any phone/tablet device I've tried it doesn't make much sense to leave this open. (I ran unit tests in Chrome, everything passed.  Did not run IE).

I would consider all Windows tablets capable of running WebGL as supported.
